data/json/recipes/armor/arms.json

```diff
@@ -7,11 +7,11 @@
     "subcategory": "CSC_ARMOR_ARMS",
     "skill_used": "fabrication",
     "difficulty": 1,
-    "time": "5 m",
+    "time": "15 m",
     "autolearn": true,
     "using": [ [ "cordage_short", 2 ] ],
     "qualities": [ { "id": "CUT", "level": 1 } ],
-    "components": [ [ [ "2x4", 1 ] ], [ [ "rag", 1 ] ] ]
+    "components": [ [ [ "2x4", 2 ] ], [ [ "rag", 1 ] ] ]
```

Comment:
Use of a single plank is intended here (as going by weight). Well, unless you also change wooden arm guards to weigh more than 2.6 kg to demand two planks, but this seems unlikely to me.

Reply:
fixed